smarts/core/scenario.py

```diff
@@ -526,16 +532,17 @@ def discover_missions_of_traffic_histories(self):
 
         return vehicle_missions
 
-    def discover_traffic_histories(self):
-        path = os.path.join(self._root, "traffic_histories.pkl")
+    @staticmethod
+    def discover_traffic_histories(scenario_root):
```

Comment:
My only last thought around this is that the traffic histories could be very large but that might be an issue for another time.